Bookish/Controllers/BookController.cs

```diff
@@ -23,8 +23,15 @@ public IActionResult AddBook()
     [HttpPost]
     public ActionResult AddBook(BookModel newBook)
     {
-        BookEditService.InsertBook(newBook);
-        return View(newBook);
+        if (BookSearchService.IsFormBlank(newBook))

```

Comment:
This returns false for any empty form since Publication Year will be 0, not null.

Reply:
I've changed the property in BookModel to be nullable and it seems to prevent this - and also therefore prevents the creation of an empty book.